### DISCUSSION TITLE
Should we include Octicons?

### DISCUSSION THREAD

**Discussion by simurai** [primer/github-vscode-theme#305] [Ideas]
Next to file icons, VS Code now (`1.45`) allows to have custom icons for the rest of the UI, like the sidebar or status bar. They call it ["product icons"](https://code.visualstudio.com/updates/v1_45#_product-icon-themes). We might can consider using the [Octicons V2](https://primer.style/octicons-v2/) for it.

- Not sure if it can be included with this theme or if it needs to be a separate one.
- It seems that we would need to convert it to an "icon font".
- We might need to make additional icons just for VS Code.

/cc @ashygee @colebemis

**Comment by edokoa** [comment]
I'd be happy about it
